test/core/networking/websockets/middleware/encryption_middleware_test.dart

```diff
@@ -0,0 +1,229 @@
+import 'dart:io';
+
+import 'package:test/test.dart';
+
+import 'package:flyde/core/networking/websockets/middleware/middleware_types.dart';
+import 'package:flyde/core/networking/websockets/middleware/encryption_middleware.dart';
+import 'package:flyde/core/networking/websockets/session.dart';
+
+import '../../../../helpers/value_hook.dart';
+import '../../../../helpers/get_uri.dart';
+import '../../../../helpers/open_webserver.dart';
+
+void main() {
+  test('EncryptionMiddleware can handshake', () async {
+    final receivedMessage = VHook<String?>(null);
+
+    final List<MiddlewareFunc> middleware = [encryptionMiddleware];
```

Comment:
type, and not only here

Reply:
The type is perfectly valid here. The first one is omitted because of the constructor. The second one is used to lock down the list.